data/p10/rw_attributes_list

```diff
@@ -0,0 +1,49 @@
+ATTR_CP_REFCLOCK_SELECT
```

Comment:
Who is a consumer of this file and How they will get it? is it consumed by pdata?